src/main/scala/workflow/graph/LabelEstimator.scala

```diff
@@ -0,0 +1,108 @@
+package workflow.graph
+
+import org.apache.spark.rdd.RDD
+
+/**
+ * A LabelEstimator has a `fitRDDs` method which takes input data and input labels,
```

Comment:
I'm not sure if this comment belongs here. Does a user who extends `LabelEstimator` need to implement this non-type-safe thing?